RubeesEat/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM mcr.microsoft.com/dotnet/sdk:8.0 AS build-env
+WORKDIR /App
+ 
```

Comment:
Please add `EXPOSE 8080` to document that the image will listen to port 8080 TCP by default.

Reply:
We tried adding a port with this comment. Then we had to set a port number in Docker Desktop to run it. But it didn't work

Reply:
Never mind, now it works.
 we didn't use 8080 before, I guess that was the issue